flink-autoscaler/src/main/java/org/apache/flink/autoscaler/config/AutoScalerOptions.java

```diff
@@ -222,6 +222,18 @@ private static ConfigOptions.OptionBuilder autoScalerConfig(String key) {
                     .withDescription(
                             "Processing rate increase threshold for detecting ineffective scaling threshold. 0.1 means if we do not accomplish at least 10% of the desired capacity increase with scaling, the action is marked ineffective.");
 
```

Comment:
I'm wondering whether we should add an on/off switch in case there are any issues with the TM metric collection.

Reply:
I could modify the logic to only query the metrics if the threshold is below 1, in any case TM metric collection issues should not block other autoscaler functionality, I have to double check that.